src/features/activity/ActivityScreen.tsx

```diff
@@ -1,25 +1,31 @@
 import React from "react";
-import { ParentPageLayout } from "@/components/ParentPageLayout";
-import { ScrollView, View } from "react-native";
+import { ScrollView, StyleSheet, View } from "react-native";
 import { ItemStack } from "@/components/Item";
-import { Layouts } from "@/theme/layouts";
+import { Layouts, SPACE } from "@/theme/layouts";
 import { mockActivity } from "@/features/activity/mocks";
 
 const ActivityScreen = () => {
   return (
-    <ParentPageLayout>
+    <View style={styles.wrapper}>
```

Comment:
I was a bit confused by the `ParentPageLayout` myself. Lots clearer this way IMHO!